Cargo.toml

```diff
@@ -49,6 +50,8 @@ members = [
     "dns-server",
     "dns-server-api",
     "end-to-end-tests",
+    "ereport/api",
```

Comment:
nit: seems like it'd be nice to be consistent about `ereport` vs. `ereporter`.  Is there a reason they're different?

Reply:
This was perhaps due to me being unnecessarily cutesy with it. I was expecting to use stuff from the `-types` crate anywhere that we deal with ereports, including in the collector and eventually diagnosis engines, while the things that actually implement the APIs are _reporters_, thus "ereporter". I can change them to be consistent.

Reply:
Made naming consistent in b70751d98c98338cde209e6ab518e9005834d3dd.